frontend/layouts/project-page/overview/component.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
```

Comment:
I kinda like all these components all under the same directory since they are all exclusive to the project page, though to keep the HeCo project directory consistent, I'd put them inside `containers` instead of `layouts`.

This is mostly because at the moment, we're only keeping the general "layout" (header, footer, content wrapper) under the `layouts` directory.